src/chat/botman.ts

```diff
@@ -1,31 +1,35 @@
 import axios from 'axios';
+import { IAttachment, IMessage } from './../typings';
 
 class BotMan {
 
-    setUserId(userId) {
+	userId: string;
+	chatServer: string;
+
+    setUserId(userId: string) {
         this.userId = userId;
     }
 
-    setChatServer(chatServer) {
+    setChatServer(chatServer: string) {
         this.chatServer = chatServer;
     }
 
-    callAPI = (text, interactive = false, attachment = null, perMessageCallback, callback) => {
+    callAPI = (text: string, interactive = false, attachment: IAttachment = null, perMessageCallback: Function, callback: Function) => {
     	let data = new FormData();
-    	const postData = {
+    	const postData: { [index: string] : string|Blob } = {
     		driver: 'web',
     		userId: this.userId,
     		message: text,
-    		attachment,
-    		interactive
+    		attachment: attachment as Blob,
+    		interactive: interactive ? '0' : '1' // FIXME: 
```

Comment:
FormData.append() expects either a String or a Blob as parameter, so I manually converted the boolean to a string.
You may want to check if I didn't break it.